frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/block-pool.ts

```diff
@@ -13,8 +13,6 @@ export const poolMessage: {
 } = {
   [POOL_PROGRESS.FAILED]: `Pool "${poolName}" already exists`,
   [POOL_PROGRESS.CREATED]: `Pool ${poolName} was successfully created`,
-  [POOL_PROGRESS.NOTALLOWED]:
-    "Pool management tasks are not supported for default pool and OpenShift Container Storage's external mode.",
   [POOL_PROGRESS.BOUNDED]: `${poolName} cannot be deleted. When a pool is bounded to PVC it cannot be deleted. Please detach all the resources from StorageClass(es): ${scName}`,
```

Comment:
No extra page for deletion, I disabled the kabab menu for external and default pool